### DISCUSSION TITLE
Tools for testing RBA

### DISCUSSION THREAD

**Discussion by ZachTheSplunker** [splunk/rba#24] [Q&A]
>Question from "The Office" Hours meetup

Does anybody have an automated tool that generates simulated examples of something in the risk index (events, system calls, etc.) for testing RBA?

**Comment by ZachTheSplunker** [comment]
>Answer from "The Office" Hours meetup

## Splunk attack Range
- https://www.splunk.com/en_us/blog/security/introducing-splunk-attack-range-v2-0.html
- https://github.com/splunk/attack_range

## Splunk Salo
- https://www.splunk.com/en_us/blog/security/introducing-synthetic-adversarial-log-objects-salo.html
- https://github.com/splunk/salo

**Comment by L0ranCyb** [comment]
https://github.com/whikernel/evtx2splunk : Ingest EVTX into a Splunk Instance
